pkg/kubectl/bind/plugin/bind.go

```diff
@@ -167,7 +166,20 @@ func (b *BindOptions) Run(ctx context.Context, urlCh chan<- string) error {
 
 	sessionID := rand.String(rand.IntnRange(20, 30))
 
-	if err := b.authenticate(provider, auth.Endpoint(ctx), sessionID, urlCh); err != nil {
+	ns, err := kubeClient.CoreV1().Namespaces().Get(ctx, "kube-bind", metav1.GetOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
+		return err
+	} else if apierrors.IsNotFound(err) {
+		ns = &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "kube-bind",
+			},
+		}
+		if ns, err = kubeClient.CoreV1().Namespaces().Create(ctx, ns, metav1.CreateOptions{}); err != nil {
+			return err
+		}
+	}
+	if err := b.authenticate(provider, auth.Endpoint(ctx), sessionID, string(ns.UID), urlCh); err != nil {
```

Comment:
neat idea about `ns.UID`. not sure about the guarantee of a global conflict for all UIDs but good enough for now.

Reply:
we still use the oidc auth id. So it only has to be unique for that user.